sites/kit.svelte.dev/src/routes/+page.svelte

```diff
@@ -1,110 +1,190 @@
-	import Logotype from './svelte-kit-logotype.svg';
-	import Machine from './svelte-kit-machine.webp?w=1440;800;600;480&format=avif;webp;png&picture';
+	import Section from '@sveltejs/site-kit/components/Section.svelte';
+	import Features from './home/Features.svelte';
+	import Hero from './home/Hero.svelte';
+	import Showcase from './home/Showcase.svelte';
+	import Try from './home/Try.svelte';
+	import Video from './home/Video.svelte';
+	import './home/common.css';
+	import Deployment from './home/Deployment.svelte';
 </script>
 
 <svelte:head>
-	<title>SvelteKit • The fastest way to build Svelte apps</title>
+	<title>SvelteKit • Web development, streamlined</title>
 
 	<meta name="twitter:title" content="SvelteKit" />
-	<meta name="twitter:description" content="The fastest way to build Svelte apps" />
+	<meta name="twitter:description" content="Web development, streamlined" />
 	<meta name="description" content="SvelteKit is the official Svelte application framework" />
 </svelte:head>
 
-<h1 class="visually-hidden">SvelteKit</h1>
-<Hero
-	title="SvelteKit"
-	logotype={Logotype}
-	tagline="The fastest way to build svelte apps"
-	background={Machine}
-	alt="SvelteKit illustration"
-/>
-
-<div class="blurb-shifter">
-	<Blurb>
-		<div slot="one">
-			<h2>Powered by Svelte</h2>
-			<p>
-				SvelteKit is an application framework powered by Svelte — build bigger apps with a smaller
-				footprint
-			</p>
-
-			<a href="https://svelte.dev" class="cta">learn Svelte</a>
+<div class="home">
+	<h1 class="visually-hidden">SvelteKit</h1>
+
+	<Hero />
+
+	<Section --background="var(--sk-back-1)">
+		<div class="grid" style="--columns: 3">
+			<div>
+				<h2>fast</h2>
+				<p>
+					Powered by <a href="https://svelte.dev">Svelte</a> and
+					<a href="https://vitejs.dev">Vite</a>, speed is baked into every aspect of SvelteKit's
+					design: fast installation, fast development, fast builds, fast page loads, fast
+					navigations. Did we mention it's fast?
+				</p>
+			</div>
+
+			<div>
+				<h2>fun</h2>
+				<p>
+					No more wasted days figuring out bundler configuration and routing and server-side
+					rendering and progressive enhancement and CSP and environment variables and all the other
+					boring stuff. Code with joy.
+				</p>
+			</div>
+
+			<div>
+				<h2>flexible</h2>
+				<p>
+					SPA? MPA? SSR? SSG? WTF. SvelteKit doesn't force its opinions on you, but instead gives
+					you the tools to succeed whatever it is you're building. And it runs wherever JavaScript
+					does.
+				</p>
+			</div>
 		</div>
-
-		<div slot="two">
-			<h2>Best of both worlds</h2>
-			<p>
-				All the SEO and progressive enhancement of a server-rendered app, with the slick navigation
-				of an SPA
-			</p>
-
-			<a data-sveltekit-preload-data href="/docs" class="cta">read the docs</a>
+	</Section>
+
+	<Try />
+
+	<Section --background="var(--background-1)">
+		<p class="definition">
+			<em>/ˈsvɛlt/</em> <span class="adjective">adjective</span>
```

Comment:
Axe flagged color contrast issues on the adjective element

> Element has insufficient color contrast of 2.28 (foreground color: #ffffff, background color: #42b4ff, font size: 12.0pt (16px), font weight: normal). Expected contrast ratio of 4.5:1

Reply:
This is another case where my inclination is to overrule Axe — it's a small throwaway detail that doesn't really affect anything, but if we deviate from the colour theme used across the site it will look out of place